### DISCUSSION TITLE
AMD 5700XT Driver Crash when playing osu!

### DISCUSSION THREAD

**Discussion by SoffiMeower** [ppy/osu#21218] [Q&A] [answered]
### Type

Crash to desktop

### Bug description

I've noticed that periodically, I'll have it where my screens freeze, but I can interact with the game via keyboard and sound cues before the entire OS locks up. (Num lock and Caps Lock don't change, nothing happens with windows keybinds), opening up Event Viewer after reboot shows Watchdog Fault Buckets (Crash dumps, essentially). WhoCrashed when reading the crashdumps, says the error is a "Video related crash". This can happen anywhere in the game.

### Screenshots or videos

![image](https://user-images.githubusercontent.com/45863583/201487955-83aaa950-62af-4802-9df4-50b88fb6db2a.png)
![image](https://user-images.githubusercontent.com/45863583/201487961-22a2b713-711c-479f-b794-d7f031d5fbed.png)
![image](https://user-images.githubusercontent.com/45863583/201487965-d40ed65f-4715-46e7-9217-58973eb07d18.png)

### Version

2022.1101.0-lazer

### Logs

[runtime.log](https://github.com/ppy/osu/files/9995582/runtime.log)
[network.log](https://github.com/ppy/osu/files/9995583/network.log)
[database.log](https://github.com/ppy/osu/files/9995584/database.log)
[performance.log](https://github.com/ppy/osu/files/9995585/performance.log)
[updater.log](https://github.com/ppy/osu/files/9995586/updater.log)
[input.log](https://github.com/ppy/osu/files/9995587/input.log)

**Comment by bdach** [comment, marked as answer]
Not much to do here for now. Most likely better to get DirectX done rather than spend time on looking into this one specific crash. The hope is that DirectX or whichever other new graphics backend will fix it.

**Reply by SoffiMeower** [reply]
What's the current GPU backend?

**Reply by bdach** [reply]
We're using opengl right now.

**Reply by SoffiMeower** [reply]
Ah, that explains it... Yeah, either Vulkan or DirectX would work. Vulkan preferred as it usually does a little better than DirectX. (And is a lot more cross-platform kind)

**Reply by SoffiMeower** [reply]
Seeing as DirectX is windows focused, Vulkan is a global GPU backend.
